cypress/integration/cucumber/step_definitions/refapp-3.x/05-vitals-and-triage/vitals-and-triage.js

```diff
@@ -0,0 +1,103 @@
+import { Given, Before, When } from "cypress-cucumber-preprocessor/steps";
+
+let identifier = null;
+let patient = null;
+
+Before({ tags: "@vitals-and-triage" }, () => {
+  cy.generateIdentifier().then((generatedIdentifier) => {
+    identifier = generatedIdentifier;
+    cy.createPatient(identifier).then((generatedPatient) => {
+      patient = generatedPatient;
+      cy.startFacilityVisit(patient.uuid);
+    });
+  });
+});
+
+Given("the user is logged in", () => {
+  cy.on("uncaught:exception", (err, runnable) => {
+    console.log(err);
+    return false;
+  });
+  cy.login();
+});
+
+Given("the user arrives on a patient’s summary page", () => {
+  cy.visit(`patient/${patient.uuid}/chart`);
+});
+
+When("the user clicks on Edit patient details", () => {
+  cy.contains("Actions").click({ force: true });
+  cy.contains("Edit patient details").click({ force: true });
+});
+
+Then("the edit page should load", () => {
+  cy.contains("Edit Patient");
+});
+
+When("the user updates the address", () => {
+  cy.get("#address1").clear({ force: true }).type("45 baker's street");
+  cy.get("#country").clear({ force: true }).type("Germany");
+  cy.get("#stateProvince").clear({ force: true }).type("Central");
+  cy.get("#cityVillage").clear({ force: true }).type("Kandy");
+  cy.get("#postalCode").clear({ force: true }).type("3000");
+  cy.contains("Update Patient").click({ force: true });
+});
+
+Then("the address should be updated", () => {
+  cy.contains("Update Patient").click({ force: true });
+});
+
+When("the user clicks on Record Vitals", () => {
+  cy.contains("Record Vitals").click({ force: true });
+});
+
+Then("the Vitals form should load", () => {
+  cy.contains("Record Vitals and Biometrics");
+});
+
+When("the user adds vitals", () => {
+  cy.get("#systolic").type("120", { force: true });
+  cy.get("#diastolic").type("80", { force: true });
+  cy.get("#Pulse").type("80", { force: true });
+  cy.get("#Notes").type("This is a note", { force: true });
+  cy.get("input[name='Oxygen Saturation']").type("90", { force: true });
+  cy.get("input[name='Respiration Rate']").type("15", { force: true });
+  cy.get("#Temperature").type("38", { force: true });
+  cy.get("#Height").type("160", { force: true });
+  cy.get("#Weight").type("60", { force: true });
+  cy.get("#MUAC").type("100", { force: true });
+});
+
+Then("the vitals form needs to be filled", () => {
+  cy.get("#systolic").should("have.value", "120");
+  cy.get("#diastolic").should("have.value", "80");
+  cy.get("#Pulse").should("have.value", "80");
+  cy.get("#Notes").should("have.value", "This is a note");
+  cy.get("input[name='Oxygen Saturation']").should("have.value", "90");
+  cy.get("input[name='Respiration Rate']").should("have.value", "15");
+  cy.get("#Temperature").should("have.value", "38");
+  cy.get("#Height").should("have.value", "160");
+  cy.get("#Weight").should("have.value", "60");
+  cy.get("#MUAC").should("have.value", "100");
+});
```

Comment:
I don't think we need to check the fields again because we don't save them right at the moment.

Reply:
Line 99 would be the ideal place to check whether the values have been saved.

Reply:
then I have to use a new form for the abnormal test right

Reply:
for line 99. Check for whatever the values should be there at that moment.